services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -151,6 +155,77 @@ async def begin_service_destruction(self, dynamic_sidecar_endpoint: str) -> None
             log_httpx_http_error(url, "POST", traceback.format_exc())
             raise e
 
+    async def service_save_state(self, dynamic_sidecar_endpoint: str) -> None:
+        url = get_url(dynamic_sidecar_endpoint, "/v1/containers/state:save")
+        try:
+            async with httpx.AsyncClient(timeout=self._save_restore_timeout) as client:
+                response = await client.post(url)
+            if response.status_code != 204:
```

Comment:
MINOR: i like using ``status. ... `` SEE https://github.com/encode/starlette/blob/master/starlette/status.py
And can be imported from fastapi as well

Reply:
fixed in the entire module